### PR TITLE
Add encrypt endpoint

### PR DESCRIPTION
**Pull request by amitaibu**
#7

### CODE REVIEW

File: routes/encrypt.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+var express = require('express');
+var router = express.Router();
+var Promise = require('bluebird');
+var request = require('request-promise');
```
Comment (amitaibu) [inline routes/encrypt.js]: remove

File: routes/encrypt.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+var express = require('express');
+var router = express.Router();
+var Promise = require('bluebird');
```
Comment (amitaibu) [inline routes/encrypt.js]: remove

**Comment by solker** [comment]
@amitaibu ready for code review.

I didn't implement a test of input parameters because request is come from drupal.

selection_094

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**